Functions/Import/xASL_imp_DCM2NII_Subject_StartConversion.m

```diff
@@ -38,18 +38,17 @@
         %% Check if we have a nii(gz) file, or something that needs to be converted (parrec/dicom)
         if ~exist(scanpath, 'dir') && ~isempty(regexpi(scanpath,'(\.nii|\.nii\.gz)$'))
             %% We found a NIfTI file, check if output exists
-            first_match = fullfile(destdir, [scan_name '.nii']);
-            if x.modules.import.imPar.bOverwrite || ~xASL_exist(first_match,'file')
-                [~, fname, fext] = fileparts(scanpath);
-                destfile = fullfile(destdir, [fname fext]); % will be renamed later
-                xASL_Copy(scanpath, destfile, x.modules.import.imPar.bOverwrite, x.modules.import.imPar.bVerbose);
-                % gunzip if required
-                destfile = xASL_adm_UnzipNifti(destfile);
-                xASL_Move(destfile, first_match, x.modules.import.imPar.bOverwrite, x.modules.import.imPar.bVerbose);
+            first_match = fullfile(destdir, [scan_name '.nii.gz']);
+            % only copy this file if it doesn't exist yet, or if we want to
+            % overwrite
+            if x.modules.import.imPar.bOverwrite || ~xASL_exist(first_match,'file') % this will check both .nii & .nii.gz
+                xASL_Copy(scanpath, first_match, x.modules.import.imPar.bOverwrite, x.modules.import.imPar.bVerbose);
+                % gzip if required
```

Comment:
Original version was unziping, now we are zipping - is that really what we want?

Reply:
Yes, we don't want to unzip, when converting to BIDS, we need to zip. So I changed it in a way that `rawdata` will always contains `.nii.gz` (I think a few lines ahead, we run `xASL_adm_GzipAllFiles` to ensure that all are zipped, so this just saves time).